src/testing

```diff
@@ -0,0 +1,111 @@
+#include <iostream>
+#include <vector>
+#include <algorithm>
+
+using namespace std;
+
+class Employee {
+public:
+    string id;
+    string name;
+
+    Employee(string empId, string empName) : id(empId), name(empName) {}
+};
+
+class EmployeeManager {
+private:
+    vector<Employee> employees;
+
+public:
+    void enterEmployeeId() {
+        string empId;
+        cout << "Enter Employee ID: ";
+        cin >> empId;
+
+        auto it = find_if(employees.begin(), employees.end(), [empId](const Employee& emp) {
+            return emp.id == empId;
+        });
+
+        if (it != employees.end()) {
+            cout << "Employee ID already exists!" << endl;
+        } else {
+            cout << "Employee ID is available." << endl;
+        }
+    }
+
+    void createEmployee() {
+        string empId, empName;
+        cout << "Enter Employee ID: ";
+        cin >> empId;
+
+        auto it = find_if(employees.begin(), employees.end(), [empId](const Employee& emp) {
+            return emp.id == empId;
+        });
+
+        if (it != employees.end()) {
+            cout << "Employee with this ID already exists!" << endl;
+            return;
+        }
+
+        cout << "Enter Employee Name: ";
+        cin.ignore(); // Clear newline character from the buffer
+        getline(cin, empName);
+
+        Employee newEmployee(empId, empName);
+        employees.push_back(newEmployee);
+
+        cout << "Employee created successfully!" << endl;
+    }
+
+    void removeEmployee() {
+        string empId;
+        cout << "Enter Employee ID to remove: ";
+        cin >> empId;
+
+        auto it = find_if(employees.begin(), employees.end(), [empId](const Employee& emp) {
+            return emp.id == empId;
+        });
+
+        if (it != employees.end()) {
+            employees.erase(it);
+            cout << "Employee removed successfully!" << endl;
+        } else {
+            cout << "Employee with this ID does not exist!" << endl;
+        }
+    }
+};
+
+int main() {
+    EmployeeManager employeeManager;
+
+    while (true) {
```

Comment:
What if I Press any other buttons on keyboard, In my case i am using a Linux machine if I press CTRL + C then it will crash the program so we need to capture the keyboard Interrupt and handle It.